pennylane/workflow/qnode.py

```diff
@@ -183,8 +183,9 @@ class QNode:
             Only applies if the device is queried for the gradient; gradient transform
             functions available in ``qml.gradients`` are only supported on the backward
             pass. The 'best' option chooses automatically between the two options and is default.
-        cache (bool or dict or Cache): Whether to cache evaluations. This can result in
-            a significant reduction in quantum evaluations during gradient computations.
+        cache=None (None or bool or dict or Cache): Whether to cache evalulations.
+            ``None`` indicates to cache only when ``max_diff > 1``. This can result in
+            a reduction in quantum evaluations during higher order gradient computations.

```

Comment:
@albi3ro minor but I wonder if `cache="auto"` is a more descriptive default here?

Reply:
switching to `"auto"`